Jenkinsfile

```diff
@@ -0,0 +1,55 @@
+@Library('mdblp-library') _
+
+pipeline {
+    agent {
+        label 'blip'
+    }
+    environment {
+        node_version='10.15'
+    }
+    stages {
+        stage('Build & Test') {
```

Comment:
It's a long stage, can you split the test & build?
For testing only: lint, test & security-checks